raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -262,33 +277,33 @@ private void maybeBecomeLeader(CandidateState state) throws IOException {
     }
 
     private void becomeCandidate() throws IOException {
-        electionTimer.reset(electionTimeoutMs + randomElectionJitterMs());
+        // after we become candidate, we may not fetch from leader any more
```

Comment:
What's the purpose of this comment?

Reply:
To explain why we reset the timer to election.